UnitySDK/Assets/ML-Agents/Scripts/Sensor/CameraSensorComponent.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using UnityEngine;
+
+namespace MLAgents.Sensor
+{
+    class CameraSensorComponent : SensorComponent
```

Comment:
I think this needs to have a list of Sensors (Since currently we support multiple cameras per agents.

Reply:
Couldn't you just add multiple CameraSensorComponents in that case? Seems easier to keep them 1:1 on the Component.

Reply:
I do not think it is possible to attach multiple of the same component to a GameObject in Unity. If there is a way then yes, makes more sense.

Reply:
![image](https://user-images.githubusercontent.com/6877802/67227159-7f4b0d80-f3eb-11e9-94fc-7c5792b4b89a.png)

Reply:
You can have multiple components with the same type.  And then access them with the `GetComponents` API.